PixiEditor/Helpers/SelectionHelpers.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using PixiEditor.Models.DataHolders;
+using PixiEditor.Models.Enums;
+using PixiEditor.Models.Position;
+using PixiEditor.Models.Undo;
+
+namespace PixiEditor.Helpers
+{
+    public class SelectionHelpers
+    {
+        public static void SelectionSet(object[] arguments)
```

Comment:
Change the name to SetSelectionProcess, generally keep the names for undo methods using OriginalName + Process